examples/apps/cra-demo/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluid-example/cra-demo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fluid-experimental/data-objects": "^0.35.0",
+    "@fluid-experimental/fluid-static": "^0.35.0",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "22.2.3",
+    "@types/node": "^10.17.24",
+    "@types/react": "^16.9.15",
+    "@types/react-dom": "^16.9.4",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
+    "react-scripts": "4.0.2",
+    "typescript": "~4.1.3",
```

Comment:
so are these?

Reply:
seems CRA with typescript just lumps everything in deps